chat-client/src/contracts/uiContracts.ts

```diff
@@ -1,13 +1,27 @@
+export interface ReferenceTrackerInformation {
```

Comment:
Do you want to duplicate these contracts here and in server? I would want to reuse those types.